stl/inc/atomic

```diff
@@ -1191,302 +1299,310 @@ struct _Atomic_integral; // not defined
 template <class _Ty>
 struct _Atomic_integral<_Ty, 1> : _Atomic_storage<_Ty> { // atomic integral operations using 1-byte intrinsics
     using _Base = _Atomic_storage<_Ty>;
+    using _TVal = typename _Base::_TVal;

```

Comment:
```suggestion
    using _Base::_TVal;
```

Reply:
had to keep `typename`, though removed `_TVal = `